cextern/trim_wcslib.sh

```diff
@@ -4,10 +4,14 @@
 
 # This removes extra large files from wcslib that aren't needed.
 
+rm -rf wcslib/C/cyril
+rm -rf wcslib/C/flexed/RCS
+rm -rf wcslib/C/RCS
```

Comment:
Is this something new?

Reply:
Oh, after reading https://github.com/astropy/astropy/blob/40e10880559dcd2dc5dd002c496684017db81b34/cextern/wcslib/CHANGES it's clear.